pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -45,12 +47,17 @@ Examples:
 				config, err := client.BuildConfigs(namespace).Get(args[0])
 				checkErr(err)
 
-				newBuild = util.GenerateBuildFromConfig(config, nil, nil)
+				//newBuild = util.GenerateBuildFromConfig(config, nil, nil)
+				imageRepos, err := client.ImageRepositories(kapi.NamespaceAll).List(labels.Everything(), labels.Everything())
```

Comment:
This is going to fail as soon as we turn on access control - you won't even be able to make this call.  You're going to need to switch to fetching the repos referenced in the triggers.